saleor/graphql/checkout/mutations/checkout_billing_address_update.py

```diff
@@ -53,11 +74,17 @@ def perform_mutation(
             error_class=CheckoutErrorCode,
         )
 
+        address_validation_rules = validation_rules or {}
```

Comment:
Maybe we could set `{}` as the default value of `validation_rules` instead of `validation_rules=None,` and we could remove this line.

Reply:
We decide that we will leave this in current format